src/frontend/Ast_to_Mir.ml

```diff
@@ -339,6 +341,16 @@ let rec param_size transform sizedtype =
   | Simplex ->
       shrink_eigen (fun d -> Expr.Helpers.(binop d Minus (int 1))) sizedtype
   | CholeskyCorr | Correlation -> shrink_eigen k_choose_2 sizedtype
+  | StochasticRow -> (
+      match sizedtype with
+      | SMatrix (mem_pattern, d1, d2) ->
+          SMatrix (mem_pattern, d1, Expr.Helpers.(binop d2 Minus (int 1)))
+      | _ -> sizedtype)
+  | StochasticColumn -> (
+      match sizedtype with
+      | SMatrix (mem_pattern, d1, d2) ->
+          SMatrix (mem_pattern, Expr.Helpers.(binop d1 Minus (int 1)), d2)
+      | _ -> sizedtype)
```

Comment:
I think this probably needs a helper like the `shrink_eigen_mat` above so that arrays of matrices work correctly?

Reply:
Worth adding an array of these to the test, as well!